exporter/datadogexporter/internal/hostmetadata/internal/ec2/ec2.go

```diff
@@ -55,28 +55,22 @@ func isDefaultHostname(hostname string) bool {
 
 // GetHostInfo gets the hostname info from EC2 metadata
 func GetHostInfo(logger *zap.Logger) (hostInfo *HostInfo) {
-	sess, err := session.NewSession()
 	hostInfo = &HostInfo{}
 
+	cfg, err := awsConfig.LoadDefaultConfig(context.Background())
```

Comment:
Does CWA rely on this datadog exporter?
Would leaving it unchanged make future upstream syncing easier?

Reply:
No required. I changed this everywhere I saw it. I can delete this in future if need be. Ideally for datadog cx they should do the same.